src/db/tables/qa_customs.ts

```diff
@@ -135,12 +135,17 @@ async function addQaCustomBroker(
   }
 }
 
-async function getAllMockedScanResults(): Promise<
-  OnerepScanResultDataBrokerRow[]
-> {
-  const res = (await knex("qa_custom_brokers").select(
-    "*",
-  )) as OnerepScanResultDataBrokerRow[];
+async function getAllMockedScanResults(
+  onerepScanId: number | null,
```

Comment:
Why is this scan id instead of profile ID?
Was this using scan ID before?

Reply:
Is this the QA table or the real table?

This is really confusing. If this is for the QA table, we can either rename or add the right column.